docs/content/concepts/storage.mdx

```diff
@@ -19,6 +19,12 @@ The exact amount of storage capacity an account has is the amount of FLOW the ac
 
 Any account can increase the storage capacity of an account by depositing more FLOW to the account.
 
+If the accounts balance is below the minimum account balance, the account's storage capacity is considered to be 0.
+
+### Storage Capacity of the Payer
+
+The storage capacity of the Payer of a transaction is generally computed the same way as the capacity of any other account, however the system needs to account for the transaction fees the payer will incur at the end of the transaction. The final amount of transaction fees is not yet fully know at the step when accounts are being checked for storage compliance (if their storage used is below their storage capacity), because of this the payers balance is conservatively considered to be lower by the maximum possible transaction fees when checking for storage compliance. The maximum possible transaction fees for a specific transaction are the transaction fees as if the transaction would have used up all of its execution effort limit.
```

Comment:
```suggestion
The storage capacity of the Payer of a transaction is generally computed the same way as the capacity of any other account, however the system needs to account for the transaction fees the payer will incur at the end of the transaction. The final amount of transaction fees is not yet fully known at the step when accounts are being checked for storage compliance (if their storage used is below their storage capacity). Because of this, the payer's balance is conservatively considered to be lower by the maximum possible transaction fees, when checking for storage compliance. The maximum transaction fee represents cost of maximum possible resource use of a transaction.
```

Reply:
> The maximum transaction  fee represents cost of maximum possible resource use of a transaction.

Yes, but the inclusion fee for a specific transaction is constant. Only the execution fees vary once the transaction is created. I wanted to make that a bit clearer. 

I changed the wording a bit. Please check again.